DTS-STN/canadian-dental-care-plan
# Removing unused application type mock and service lookup

### Description
The "Type of application" page is using i18n files to display the radio option labels. The only way to continue is if the user selects "I am applying for myself". There is no reason for these applicant type mocks/service. 

When the entire application is submitted, we can define and use a new environment variable that represents the ID for an "Applicant" (other options are "Dependent" and "Spouse", which aren't currently applicable).

### Checklist

### Test Instructions
Run application and verify that you can proceed after the "Type of application" page.
